seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/IncrementalSource.java

```diff
@@ -79,7 +78,7 @@
 
 @NoArgsConstructor
 public abstract class IncrementalSource<T, C extends SourceConfig>
-        implements SeaTunnelSource<T, SourceSplitBase, PendingSplitsState>, SupportCoordinate {
```

Comment:
I don't think he can be removed

Reply:
@hailin0 Please explain it.

Reply:
@TyrantLucifer  I believe that most data sources do not have the capability to make changes. appendStream is more suitable for them. SupportCoordinate can better distinguish this type of behavior.

Reply:
For the Spark engine, which does not naturally support RowKind, it is also necessary to distinguish between these two types of data sources.